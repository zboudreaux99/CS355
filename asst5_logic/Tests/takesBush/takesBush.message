Output line 0:
  <i>expected: </i>Copyright (C) 1999-2025 Daniel Diaz <span class='expected'>GNU</span> Prolog 1.5.0 (64 bits) cs311 cs450 --- end of output --- 
  <i>observed: </i>Copyright (C) 1999-2025 Daniel Diaz <span class='observed'>cs</span>311 cs450 GNU Prolog 1.5.0 (64 bits) --- end of output --- 
