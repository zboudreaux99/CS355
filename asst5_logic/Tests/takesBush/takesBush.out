Copyright (C) 1999-2025 Daniel Diaz
cs311
cs450
GNU Prolog 1.5.0 (64 bits)
